### PR TITLE
feature(ui-ux): create proof of assets section

### PR DESCRIPTION
**Pull request by lykalabrada**
#### What this PR does / why we need it:
- Added proof of assets section for web and tablet
- Added reusable daily limit to fit both web and mobile view UI
- Added bottom menu for mobile view
- Added reusable component for bottom modal on mobile

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
- Dependent of PR https://github.com/WavesHQ/bridge/pull/37
- All info icon tooltips are non-working. Will create a separate PR

#### Developer Checklist:

- [x] Read your code changes at least once
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [x] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: apps/web/src/components/ProofOfAssetsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        </button>
+      </div>
+      <div className="flex flex-wrap gap-3 lg:gap-2 mt-2">
+        <TokenSupplyItem token={selectedTokensA.tokenA} />
+        <TokenSupplyItem token={selectedTokensB.tokenA} />
+      </div>
+      <div className="hidden md:block mt-5 lg:mt-6">
+        <DailyLimit />
+      </div>
+      <div className="flex items-center border-t-[0.5px] border-t-dark-200 md:border-t-0 rounded-b-lg lg:rounded-b-xl md:dark-bg-card-section md:-mx-6 mt-5 md:mt-4 lg:mt-6 lg:-mx-8 pt-4 pb-0 md:pb-5 md:px-6 lg:px-8 lg:py-5">
```
Comment (kyleleow) [inline apps/web/src/components/ProofOfAssetsCard.tsx:48]:
```suggestion
      <div className="flex items-center border-t-[0.5px] border-t-dark-200 md:border-t-0 rounded-b-lg lg:rounded-b-xl md:dark-bg-card-section md:-mx-6 mt-5 md:mt-4 lg:mt-6 lg:-mx-8 pt-4 pb-0 md:pb-5 md:px-6 lg:px-8">
```

File: apps/web/src/components/MobileBottomMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+export default function MobileBottomMenu() {
+  const router = useRouter();
+  const [openProofOfAssets, setOpenProofOfAssets] = useState<boolean>(false);
+
+  const menuList: MenuListItem[] = [
+    {
+      id: "assets",
+      title: "Proof of Assets",
```
Comment (kyleleow) [inline apps/web/src/components/MobileBottomMenu.tsx:22]:
```suggestion
      title: "Proof of assets",
```

File: apps/web/src/components/MobileBottomMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      id: "guide",
+      title: "User Guide",
+      icon: FiBook,
+      onClick: () => router.push("/user-guide"),
+    },
+  ];
+
+  return (
+    <>
+      <ul className="flex justify-between">
```
Comment (kyleleow) [inline apps/web/src/components/MobileBottomMenu.tsx:42]: can we space each item equally? i think `grid` could fit better here

File: apps/web/src/components/MobileBottomMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    },
+  ];
+
+  return (
+    <>
+      <ul className="grid grid-cols-3 gap-2">
+        {menuList.map(({ icon: Icon, ...item }) => (
+          <li
+            key={item.title}
+            aria-hidden="true"
```
Comment (kyleleow) [inline apps/web/src/components/MobileBottomMenu.tsx:46]: we need this?

File: apps/web/src/components/ProofOfAssetsCard.tsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      <div className="flex items-center mt-5 lg:mt-6">
+        <span className="text-xs lg:text-sm font-semibold lg:tracking-wide text-dark-700">
+          TOKEN SUPPLY
+        </span>
+        <button type="button" className="ml-2 focus:outline-none">
+          {/* TODO: Disply token supply info onclick */}
+          <FiInfo size={16} className="text-dark-700" />
+        </button>
+      </div>
+      <div className="flex flex-wrap gap-3 lg:gap-2 mt-2">
```
Comment (kyleleow) [inline apps/web/src/components/ProofOfAssetsCard.tsx:41]:
```suggestion
      <div className="flex flex-wrap gap-3 lg:gap-2 mt-3 md:mt-2">
```

File: apps/web/src/components/ProofOfAssetsCard.tsx [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+function TokenSupplyItem({ token }: { token: TokenDetailI }) {
+  return (
+    <div className="flex flex-row items-center min-w-[45%] 2xl:min-w-[30%]">
+      <Image
+        width={100}
+        height={100}
+        src={token.icon}
+        alt={token.name}
+        className="w-5 h-5 lg:w-6 lg:h-6"
```
Comment (kyleleow) [inline apps/web/src/components/ProofOfAssetsCard.tsx:64]:
```suggestion
        className="w-7 h-7 md:w-5 md:h-5 lg:w-6 lg:h-6"
```

File: apps/web/src/components/MobileBottomMenu.tsx (hunk 4 above)
Comment (lykalabrada) [reply apps/web/src/components/MobileBottomMenu.tsx:46]: To easily get rid of this lint error:
image

But maybe we shoudn't do this. Let me update

Comment (lykalabrada) [reply apps/web/src/components/MobileBottomMenu.tsx:46]: Yup, I'm replacing with div as button

File: apps/web/src/components/ProofOfAssetsCard.tsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+      />
+      <NumericFormat
+        className="text-left text-dark-900 lg:text-lg leading-5 lg:leading-6 tracking-[0.01em] lg:tracking-normal ml-2 lg:ml-1"
+        value={token.supply}
+        decimalScale={4}
+        thousandSeparator
+        suffix={` ${token.name}`}
```
[the remark this answers is not in the document]
Comment (lykalabrada) [reply apps/web/src/components/ProofOfAssetsCard.tsx:71]: Thanks for catching this! Also for tablet and web it's smaller. Updating

File: apps/web/src/components/ProofOfAssetsCard.tsx [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+      <Image
+        width={100}
+        height={100}
+        src={token.icon}
+        alt={token.name}
+        className="w-7 h-7 md:w-5 md:h-5 lg:w-6 lg:h-6"
+      />
+      <span>
+        <NumericFormat
+          className="text-left text-dark-900 lg:text-lg leading-5 lg:leading-6 tracking-[0.01em] lg:tracking-normal ml-2 lg:ml-1"
```
Comment (kyleleow) [inline apps/web/src/components/ProofOfAssetsCard.tsx:68]: font size 18px on mobile

File: apps/web/src/components/MobileBottomMenu.tsx (hunk 4 above)
Comment (kyleleow) [reply apps/web/src/components/MobileBottomMenu.tsx]: sure

File: apps/web/src/components/ProofOfAssetsCard.tsx (hunk 8 above)
Comment (lykalabrada) [reply apps/web/src/components/ProofOfAssetsCard.tsx:68]: 😵‍💫 will update for mobile, tablet is 16px I believe

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**